Sources/Danger/GitHubDSL.swift

```diff
@@ -17,489 +13,449 @@ public struct GitHub: Decodable {
-        case updatedAt = "updated_at"
-        case closedAt = "closed_at"
+extension GitHub {
+    public struct Review: Decodable, Equatable {
+        enum CodingKeys: String, CodingKey {
+            case user
+            case id
+            case body
+            case state
+            case commitId = "commit_id"
+        }
+
+        public enum State: String, Decodable {
+            case approved = "APPROVED"
+            case requestedChanges = "CHANGES_REQUESTED"
+            case comment = "COMMENTED"
+            case pending = "PENDING"
+            case dismissed = "DISMISSED"
+        }
+
+        /// The user who has completed the review or has been requested to review.
+        public let user: User
+
+        /// The id for the review (if a review was made).
+        public let id: Int?
+
+        /// The body of the review (if a review was made).
+        public let body: String?
+
+        /// The commit ID the review was made on (if a review was made).
+        public let commitId: String?
+
+        /// The state of the review (if a review was made).
+        public let state: State?
     }
+}
 
-    // MARK: - IssueState
-
-    public enum IssueState: String, Decodable {
-        case open
-        case closed
-        case locked
+extension GitHub {
+    /// A GitHub specific implementation of a git commit.
+    public struct Commit: Decodable, Equatable {
+        enum CodingKeys: String, CodingKey {
+            case sha
+            case commit
+            case url
+            case author
+            case committer
+        }
+
+        /// The SHA for the commit.
+        public let sha: String
+
+        /// The raw commit metadata.
+        public let commit: Git.Commit
+
+        /// The URL for the commit on GitHub.
+        public let url: String
+
+        /// The GitHub user who wrote the code.
+        public let author: User?
+
+        /// The GitHub user who shipped the code.
+        public let committer: User?
+
+        public init(from decoder: Decoder) throws {
+            let container = try decoder.container(keyedBy: CodingKeys.self)
+
+            sha = try container.decode(String.self, forKey: .sha)
+            commit = try container.decode(Git.Commit.self, forKey: .commit)
+            url = try container.decode(String.self, forKey: .url)
+            author = (try? container.decodeIfPresent(User.self, forKey: .author)) ?? nil
+            committer = (try? container.decodeIfPresent(User.self, forKey: .committer)) ?? nil
+        }
     }
+}
 
-    // MARK: - Properties
-
-    /// The id number of the issue
-    public let id: Int
-
-    /// The number of the issue.
-    public let number: Int
-
-    /// The title of the issue.
-    public let title: String
-
-    /// The user who created the issue.
-    public let user: GitHubUser
+extension GitHub {
+    public struct Issue: Decodable, Equatable {
+        enum CodingKeys: String, CodingKey {
+            case id
```

Comment:
<!--
  0 failure: 
  1 warning:  Enum element name...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/GitHubDSL.swift;
  Line: 318;
-->

- :warning: Enum element name should be between 3 and 40 characters long: 'id' (`identifier_name`)